python/hail/api1/context.py

```diff
@@ -818,19 +818,19 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, min_p
             dataset.  Don't load sample ids, sample annotations or
             genotypes.
 
-        :param call_fields: FORMAT fields in VCF to treat as a :py:class:`~hail.type.TCall`. Only applies if ``generic=True``.
+        :param call_fields: FORMAT fields in VCF to treat as a :class:`.~hail.type.TCall`. Only applies if ``generic=True``.
         :type call_fields: str or list of str
 
         :param bool generic: If True, read the genotype with a generic schema.
         
-        :param reference_genome: Reference genome to use. Default is :class:`~.HailContext.default_reference`.
+        :param reference_genome: Reference genome to use. Default is :class:`.~.HailContext.default_reference`.
```

Comment:
does this work with two dots?

Reply:
oops, probably not. Good catch, I must have done some sloppy regex replacing